nexus_constructor/model/stream.py

```diff
@@ -1,15 +1,24 @@
+from abc import ABC
 from enum import Enum
-from typing import List, Union
+from typing import TYPE_CHECKING, List, Union
 
 import attr
+import numpy as np
 
 from nexus_constructor.common_attrs import CommonKeys, NodeType
-from nexus_constructor.model.group import Group
+from nexus_constructor.model.attributes import Attributes
+
+if TYPE_CHECKING:
+    from nexus_constructor.model.group import Group  # noqa: F401
+
+from nexus_constructor.model.value_type import ValueType
 
 ARRAY_SIZE = "array_size"
 VALUE_UNITS = "value_units"
 CHUNK_CHUNK_KB = "nexus.chunk.chunk_kb"
 CHUNK_CHUNK_MB = "nexus.chunk.chunk_mb"
+CHUNK_SIZE = "chunk_size"
+CUE_INTERVAL = "cue_interval"
 INDEX_EVERY_KB = "nexus.indices.index_every_kb"
```

Comment:
Check if these can be removed.

Reply:
They are used in the UI. So it will need to be fixed in a separate ticket which I already created:
https://jira.esss.lu.se/browse/ECDC-2725